### PR TITLE
Style news feed with material cards

### PR DESCRIPTION
**Pull request by schneider-clarisse**

### CODE REVIEW

**Comment by ericljpemberton** [comment]
Indentation is a little wacky

image

File: src/js/components/NewsFeedPost.jsx [hunk 1]
```diff
@@ -123,45 +124,31 @@ export default class NewsFeedPost extends React.Component {
...
-      );
-    }
-
     return (
-      <div className="newsfeed-post">
-        <div className="profile-photo inline">
-          <img alt="poster" src={poster_icon} className="user-photo no-selection" />
-        </div>
-        <div className="post-text inline">
-          <div className="right-options">
```
Comment (ericljpemberton) [inline src/js/components/NewsFeedPost.jsx:142]: I'm not saying that this styling should be in here but I didn't know where to comment that the associated "flag" button no longer seems to exist....

Marked under the actions variable
```
    let actions = null;
    if (owner) {
      actions = (
        <FaTrashO
          className="clickable right-icons no-selection"
          onClick={this.deletePost}
          size={20}
        />
      );
    } else {
      actions = (
        <FaFlag
          className="clickable right-icons no-selection"
          onClick={this.changeFlag}
          style={flagColor}
          size={20}
        />
      );
    }
...............
          <div className="right-options">
            {actions}
          </div>
```

File: src/js/components/NewsFeedPost.jsx [hunk 2]
```diff
@@ -123,45 +124,31 @@ export default class NewsFeedPost extends React.Component {
...
-          </div>
-        </div>
-      </div>
+	  <BetterCard>
+		<CardTitle
+		  avatar={poster_icon}
+		  title={poster_name}
+		  subtitle={moment(timestamp * 1000).format('MMMM Do, h:mm a')}/>
+		<CardText>{text}</CardText>
+		{ media_link && media_link !== '' &&
```
Comment (ericljpemberton) [inline src/js/components/NewsFeedPost.jsx]: {actions} (the location of the flagging functionality was deleted here

**Comment by ericljpemberton** [comment]
Posts with dates were a no go?

File: src/js/components/NewsFeedPost.jsx [hunk 3]
```diff
@@ -123,45 +124,31 @@ export default class NewsFeedPost extends React.Component {
...
+			onClick={this.changeLike}
+			style={heartColor}
+			size={20}
+		  />
+		  <span className="like-count no-selection">{like_count}</span>
+		  { repost &&
+			<FaRetweet className="repost-icon clickable" onClick={this.retweet} size={28} />
+		  }
+		</CardActions>
+	  </BetterCard>
```
Comment (ericljpemberton) [inline src/js/components/NewsFeedPost.jsx]: Dates/time of post = bad?

File: src/js/components/CreatePost.jsx [hunk 4]
```diff
@@ -152,88 +154,58 @@ export default class NewsFeed extends React.Component {
...
+        </CardText>
+		{ preview &&
+		  <CardMedia
+			aspectRatio="wide"
+			image={preview}>
+            <MdHighlightRemove className="photo-delete" onClick={this.deletePhoto} size={24} />
+          </CardMedia>
+        }
+		<CardActions>
+		  <Dropzone className="dropzone border" onDrop={this.onPhotoDrop} multiple={false}>
```
Comment (ericljpemberton) [inline src/js/components/CreatePost.jsx]: I'm going to assume everything still styles correctly?
Is the dropzone class still a thing? It looks like it is dropped unless there is magic scss going on or it exists elsewhere

**Comment by ericljpemberton** [comment]
Holy crap that's a lot of removed css, awesome!

File: src/js/components/NewsFeedPost.jsx (hunk 1 above)
Comment (schneider-clarisse) [reply src/js/components/NewsFeedPost.jsx:142]: right sorry that's my bad i'll put it back in

**Comment by larajanecka** [comment]
Looks like theres some funky indentation going on. NBD but `eslint --fix` _filename_ will insta fix it for ya.

**Pull request merged**